Tools/docker_run.sh

```diff
@@ -4,7 +4,7 @@ if [ -z ${PX4_DOCKER_REPO+x} ]; then
 	echo "guessing PX4_DOCKER_REPO based on input";
 	if [[ $@ =~ .*px4_fmu.* ]]; then
 		# nuttx-px4fmu-v{1,2,3,4,5}
-		PX4_DOCKER_REPO="px4io/px4-dev-nuttx:2019-11-25"
+		PX4_DOCKER_REPO="px4io/px4-dev-nuttx:latest"
```

Comment:
I think this is not a good idea since the behavior will not be consistent whenever there is a image released

Reply:
Yes we definitely want the specific tag specified. This really helps when you jump between different branches where the toolchain may have changed.